packages/cloudflare/src/api/memory-queue.ts

```diff
@@ -0,0 +1,51 @@
+import logger from "@opennextjs/aws/logger.js";
+import type { Queue, QueueMessage } from "@opennextjs/aws/types/overrides.js";
+
+type Options = { revalidationTimeoutMs: number };
+
+export const DEFAULT_REVALIDATION_TIMEOUT = 10_000;
+
+/**
+ * The Memory Queue offers basic ISR revalidation by directly requesting a revalidation of a route.
+ *
+ * It offers basic support for in-memory de-duping per isolate.
+ */
+export class MemoryQueue implements Queue {
+  readonly name = "memory-queue";
+
+  public revalidatedPaths = new Map<string, ReturnType<typeof setTimeout>>();
+
+  constructor(private opts: Options = { revalidationTimeoutMs: DEFAULT_REVALIDATION_TIMEOUT }) {}
```

Comment:
Couldn't the type be inferred here?

```suggestion
  constructor(private opts = { revalidationTimeoutMs: DEFAULT_REVALIDATION_TIMEOUT }) {}
```